pkg/databind/internal/discovery/docker/docker.go

```diff
@@ -47,7 +48,7 @@ func fetch(d discovery.Container, matcher *discovery.FieldsMatcher) ([]discovery
 	}
 	defer dc.Close()
 
-	containers, err := dc.ContainerList(context.Background(), types.ContainerListOptions{})
+	containers, err := dc.ContainerList(context.Background(), container.ListOptions{})
```

Comment:
NIT: since it's legacy, you may just use nolint: exhaustruct

Reply:
though about it but don't want to pollute the code with nolint directives TBH